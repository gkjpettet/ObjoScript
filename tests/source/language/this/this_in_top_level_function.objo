function test() {
this # Expect error.
}