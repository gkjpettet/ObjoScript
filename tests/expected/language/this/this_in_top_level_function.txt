`this` can only be used within a method or constructor.